spec/controllers/liked_jobs_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe LikedJobsController, type: :controller do
+  before do
+    @user = create(:user)
+    allow_any_instance_of(ApplicationController)
+                         .to receive(:current_user).and_return(@user)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 21) spaces for indenting an expression spanning multiple lines.